proto/minder/v1/minder.proto

```diff
@@ -1810,13 +1810,25 @@ message ListEvaluationResultsRequest {
     // an empty label_filter.
     oneof profile_selector {
         // ID can contain either a profile name or an ID
-        string profile = 2;
+        string profile = 2 [
+            (buf.validate.field).string = {
+                pattern: "^([[:alnum:]][-[:word:]]*)?$"
```

Comment:
If you put all this in `^()?$`, then you don't need `ignore`, but I'd probably remove the parenthesis and keep the ignore for clarity in this case.